frontend/src/components/table.tsx

```diff
@@ -0,0 +1,137 @@
+import React, { useState } from 'react';
+import '../styles/table.css';
+
+interface driversSettings {
+  name: string;
+  netid: string;
+  email: string;
+  phone: string;
+}
+
+interface FormProps {
+  onClick: ((newDriver: driversSettings) => void);
+}
+
+function deleteEntry(netid: string, driverList: driversSettings[]) {
+  const newDriverList = [];
+  for (let i = 0; i < driverList.length; i += 1) {
+    if (driverList[i].netid !== netid) {
+      newDriverList.push(driverList[i]);
+    }
+  }
+  return newDriverList;
+}
+
+function renderTableHeader() {
+  return (
+    <tr>
+      <th>Name</th>
+      <th>Netid</th>
+      <th>Email</th>
+      <th>Phone Number</th>
+    </tr>
+  );
+}
+
+function addDriver(newDriver: driversSettings, allDrivers: driversSettings[]) {
+  const updatedDrivers = [...allDrivers, newDriver];
+  return updatedDrivers;
+}
+
+const Form = (props: FormProps) => {
+  const $newDriver = { name: '', netid: '', email: '', phone: '' };
```

Comment:
No need to have $ before your variable names. Wherever you have a $, just remove it.